src/components/EntityPage/File/experimentalStrategies.js

```diff
@@ -0,0 +1,25 @@
+import { pickData } from './utils';
+
+export const experimentalStrategiesColumns = [
+  { Header: 'Experimental Strategy', accessor: 'experiment_strategy' },
+  { Header: 'External ID', accessor: 'external_id' },
+  { Header: 'Experiment Date', accessor: 'experiment_date' },
+  { Header: 'Instrument Model', accessor: 'instrument_model' },
+  { Header: 'Platform', accessor: 'platform' },
+  { Header: 'Library Name', accessor: 'library_name' },
+  { Header: 'Library Strand', accessor: 'library_strand' },
+];
+
+export const experimentalStrategiesData = data =>
```

Comment:
I prefer functions named with verbs or nouns prefixed with "to", reads better in my opinion